app/views/subscriptions_management/update_address.html.erb

```diff
@@ -0,0 +1,59 @@
+<% content_for :title, 'Change your email address' %>
+
+<% content_for :head do %>
+  <meta name="robots" content="noindex, nofollow">
+<% end %>
+
+<% content_for :head_css do %>
+  <%= stylesheet_link_tag 'application-without-elements' %>
+<% end %>
+
+<%= render 'govuk_publishing_components/components/back_link', {
+  href: @back_url,
+} %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title', title: 'Change your email address' %>
+  </div>
+</div>
+
+<% if flash[:error] %>
+  <div class="grid-row responsive-bottom-margin">
+    <div class="column-two-thirds">
+      <%= render 'govuk_publishing_components/components/error_summary', {
+        title: 'We weren’t able to change your email address',
+        items: [
+          {
+            text: 'There’s a problem with your email address',
+            href: '#email-address-input',
+          }
+        ]
+      } %>
+    </div>
+  </div>
+<% end %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/govspeak', {
+      content: %{<p>Your current email address is #{@address}</p>},
+      rich_govspeak: true
+    } %>
+    <%= form_tag change_address_path, method: :post do %>
+      <%= render 'govuk_publishing_components/components/input', {
+        error_message: flash[:error],
```

Comment:
Aside: I'd expect this error to render in red rather than grey (based on the screenshot). https://govuk-publishing-components.herokuapp.com/component-guide/input shows grey, so probably a component issue.